docs/guide/intro.rst

```diff
@@ -0,0 +1,107 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_intro:
+
+Introduction
+=============
+
+Welcome to discord.py, a library for Python to aid in creating applications for Discord.
+
+Prerequisites
+--------------
+
+To install discord.py you'll need Python version 3.8.0 or later.
+
+.. _guide_intro_installation:
+
+Installation
+-------------
+
+On Unix systems you can run the following command to install discord.py from PyPI.
+
+.. code-block:: shell
+
+    python3 -m pip install -U discord.py
+
+On Windows systems, you can use the following command instead.
+
+.. code-block:: shell
+
+    py -3 -m pip install -U discord.py
+
+Voice Support
+~~~~~~~~~~~~~~
+
+Voice support (e.g. joining voice channels and playing music) is not supported by default, and can be installed by installing ``discord.py[voice]`` instead of ``discord.py``; ::
```

Comment:
```suggestion
Voice support (e.g. playing audio in voice channels) is not enabled by default and can be enabled by installing ``discord.py[voice]`` instead of ``discord.py``. ::
```